backend/src/data/GuildSavedMessages.ts

```diff
@@ -35,7 +35,24 @@ export class GuildSavedMessages extends BaseGuildRepository {
     this.toBePermanent = new Set();
   }
 
-  public msgToSavedMessageData(msg: Message): ISavedMessageData {
+  public static msgToSavedMessage(message: Message): SavedMessage {
```

Comment:
Was there any reason you in particular made this a static function?

Reply:
I'd rather create a guild-specific instance of the class and put it in Automod's `state`. We can look into extracting `msgToSavedMessage` out of the GuildSavedMessages class later if necessary.

Reply:
I just read the class name. I thought it was GuildSavedMessage the whole time.
Also, I made it static because it's being used [here](https://github.com/almeidx/ZeppelinBot/blob/a3c3a44fa7d08432670cca3e6f3034f9639219bc/backend/src/plugins/Automod/events/runAutomodOnThreadEvents.ts#L25)

Reply:
@Dragory Done